src/shared/ui/SketchCanvas/LineSketcher.ts

```diff
@@ -117,6 +117,12 @@ class LineSketcher {
   public getFirstPoint(): Point {
     return this.points[0];
   }
+
+  public getLastPoint(): Point {
```

Comment:
The same applies to `getFirstPoint` function

Reply:
@vmkhitaryanscn  yeah, I got that it will be guaranteed but it depends on the implementation of the external function that calls `getLastPoint` function. As implementations tend to get changed from time to time it is beneficial to just put the right types to prevent possible errors in the future.